Gemfile.lock

```diff
@@ -1,25 +1,38 @@
 PATH
   remote: .
   specs:
-    os (1.0.1)
+    os (1.1.4)
 
 GEM
   remote: http://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    power_assert (1.0.2)
+    diff-lcs (1.5.0)
+    power_assert (2.0.1)
     rake (0.9.6)
-    test-unit (3.2.3)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-support (3.11.0)
+    test-unit (3.5.3)
       power_assert
 
 PLATFORMS
-  ruby
-  x86-mingw32
+  arm64-darwin-21
 
 DEPENDENCIES
+  os!
   rake (~> 0.8)
-  rspec (~> 2.5.0)
+  rspec (>= 2.0)
```

Comment:
Loosening here will allow using the `transpec` tool to automatically upgrade the RSpec syntax to RSpec 3+.